integration-cli/docker_cli_inspect_test.go

```diff
@@ -93,13 +88,10 @@ func (s *DockerSuite) TestInspectTypeFlagContainer(c *check.C) {
 
 	formatStr := fmt.Sprintf("--format='{{.State.Running}}'")
 	out, exitCode, err := dockerCmdWithError("inspect", "--type=container", formatStr, "busybox")
```

Comment:
Just `dockerCmd` here, no need to check exitcode/error since `dockerCmd` does this.